Users/service.js

```diff
@@ -1,51 +1,63 @@
 var response = require('../Objects/response.js');
 
-var createUser = function (json) {
-    // Creates a user
-
-    // TODO: Create user and remove console.log
-    console.log(json);
+var createUser = (json) => {
+	// Create Student
+	if (json.role == 'Student') {
+		var Student = new StudentModel({
```

Comment:
```Student``` -> ```student```.  Local variable names should be in lowerCamelCase: https://google.github.io/styleguide/jsguide.html#naming-local-variable-names